automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -94,37 +95,21 @@ class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
       val role = "bigquery.jobUser"
 
       val user: Credentials = UserPool.chooseStudent
-      val userToken: AuthToken = user.makeAuthToken()
-
-      // user is not a project owner
-      val errorMsg1 = "You must be a project owner"
-
-      withBillingProject("auto-goog-role") { projectName =>
-        val addEx = intercept[APIException] {
-          Orchestration.billing.addGoogleRoleToBillingProjectUser(projectName, user.email, role)(userToken)
-        }
-        addEx.getMessage should include(errorMsg1)
-
-        val removeEx = intercept[APIException] {
-          Orchestration.billing.removeGoogleRoleFromBillingProjectUser(projectName, user.email, role)(userToken)
-        }
-        removeEx.getMessage should include(errorMsg1)
-      }(ownerToken)
-
-      // owner is not an owner of this project
-      val errorMsg2 = "The caller does not have permission"
 
-      val otherProject = "broad-dsde-dev"
+      val errorMsg = "You must be a project owner"
+      val unownedProject = "broad-dsde-production"
 
       val addEx = intercept[APIException] {
-        Orchestration.billing.addGoogleRoleToBillingProjectUser(otherProject, user.email, role)(ownerToken)
+        Orchestration.billing.addGoogleRoleToBillingProjectUser(unownedProject, user.email, role)(ownerToken)
       }
-      addEx.getMessage should include(errorMsg2)
+      addEx.getMessage should include(errorMsg)
+      addEx.getMessage should include(StatusCodes.Forbidden.intValue.toString)
 
       val removeEx = intercept[APIException] {
-        Orchestration.billing.removeGoogleRoleFromBillingProjectUser(otherProject, user.email, role)(ownerToken)
+        Orchestration.billing.removeGoogleRoleFromBillingProjectUser(unownedProject, user.email, role)(ownerToken)
```

Comment:
This is probably fine, however I can't not point out that this request doesn't make sense because `user.email` is (hopefully!) not in broad-dsde-production.

Reply:
This is a very good point.

Reply:
Talked about this with Hussein - it does not actually matter for this functionality whether the user is a member of the google project.  Consider that we have many FC users who aren't members of *any* google project.  It does make sense to use this endpoint to give access to these users.

Reply:
My comment here was specifically about "remove". I agree that it shouldn't matter; I'd expect the code to check permissions for the actor on the project first and never even get to checking for the target user on the project. However I like to make sure to think through these implementation assumptions before building them into test code so I have confidence that the test will remain valid in the future.